Cannot find module 'webpack-sources'

<!--
IF YOU DON'T FILL OUT THE FOLLOWING INFORMATION YOUR ISSUE MIGHT BE CLOSED WITHOUT INVESTIGATING
-->
### Bug Report or Feature Request (mark with an `x`)
```
- [x] bug report -> please search issues before submitting
- [ ] feature request
```

### Versions.
<!--
Output from: `ng --version`.
If nothing, output from: `node --version` and `npm --version`.
  Windows (7/8/10). Linux (incl. distribution). macOS (El Capitan? Sierra?)
-->
@angular/cli: 1.4.7
node: 8.3.0
os: win32 x64
@angular/animations: 4.3.6
@angular/cdk: 2.0.0-beta.10
@angular/common: 4.3.6
@angular/compiler: 4.3.6
@angular/core: 4.3.6
@angular/forms: 4.3.6
@angular/http: 4.3.6
@angular/material: 2.0.0-beta.10
@angular/platform-browser: 4.3.6
@angular/platform-browser-dynamic: 4.3.6
@angular/router: 4.3.6
@angular/cli: 1.4.7
@angular/compiler-cli: 4.3.6
typescript: 2.2.2

### Repro steps.
<!--
Simple steps to reproduce this bug.
Please include: commands run, packages added, related code changes.
A link to a sample repo would help too.
-->

ng --prod --aot

### The log given by the failure.
<!-- Normally this include a stack trace and some more information. -->

Cannot find module 'webpack-sources'
Error: Cannot find module 'webpack-sources'
    at Function.Module._resolveFilename (module.js:489:15)
    at Function.Module._load (module.js:439:25)
    at Module.require (module.js:517:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (E:\PKI\std\modules\web\admin-rc\node_modules\license-webpack-plugin\dist\LicenseWebp
ackPlugin.js:14:25)
    at Module._compile (module.js:573:30)
    at Object.Module._extensions..js (module.js:584:10)
    at Module.load (module.js:507:32)
    at tryModuleLoad (module.js:470:12)
    at Function.Module._load (module.js:462:3)
    at Module.require (module.js:517:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (E:\PKI\std\modules\web\admin-rc\node_modules\license-webpack-plugin\dist\index.js:3:
30)
    at Module._compile (module.js:573:30)
    at Object.Module._extensions..js (module.js:584:10)
    at Module.load (module.js:507:32)
    at tryModuleLoad (module.js:470:12)
    at Function.Module._load (module.js:462:3)
    at Module.require (module.js:517:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (E:\PKI\std\modules\web\admin-rc\node_modules\@angular\cli\models\webpack-configs\pro
duction.js:8:34)
    at Module._compile (module.js:573:30)

after install with `ng new foo` i get 
`Cannot find module 'webpack-sources'`

running 
`ng serve`

nvm => node 6.11.1
npm 5.4.2

```
@angular/cli: 1.4.7
node: 6.11.1
os: darwin x64
@angular/animations: 4.4.5
@angular/common: 4.4.5
@angular/compiler: 4.4.5
@angular/core: 4.4.5
@angular/forms: 4.4.5
@angular/http: 4.4.5
@angular/platform-browser: 4.4.5
@angular/platform-browser-dynamic: 4.4.5
@angular/router: 4.4.5
@angular/cli: 1.4.7
@angular/compiler-cli: 4.4.5
@angular/language-service: 4.4.5
typescript: 2.3.4
```

--- a/package.json
+++ b/package.json
@@ -98,6 +98,7 @@
     "webpack-dev-middleware": "~1.12.0",
     "webpack-dev-server": "~2.7.1",
     "webpack-merge": "^4.1.0",
+    "webpack-sources": "^1.0.0",
     "webpack-subresource-integrity": "^1.0.1",
     "zone.js": "^0.8.14"
   },

--- a/packages/@angular/cli/models/webpack-configs/production.ts
+++ b/packages/@angular/cli/models/webpack-configs/production.ts
@@ -13,6 +13,13 @@ import { readTsconfig } from '../../utilities/read-tsconfig';
 
 const UglifyJSPlugin = require('uglifyjs-webpack-plugin');
 
+/**
+ * license-webpack-plugin has a peer dependency on webpack-sources, list it in a comment to
+ * let the dependency validator know it is used.
+ *
+ * require('webpack-sources')
+ */
+
 
 export function getProdConfig(wco: WebpackConfigOptions) {
   const { projectRoot, buildOptions, appConfig } = wco;

--- a/packages/@angular/cli/package.json
+++ b/packages/@angular/cli/package.json
@@ -81,6 +81,7 @@
     "webpack-dev-middleware": "~1.12.0",
     "webpack-dev-server": "~2.7.1",
     "webpack-merge": "^4.1.0",
+    "webpack-sources": "^1.0.0",
     "webpack-subresource-integrity": "^1.0.1",
     "zone.js": "^0.8.14"
   },

